connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -192,7 +192,7 @@ private GeyserConnector(PlatformType platformType, GeyserBootstrap bootstrap) {
             }
         }
 
-        defaultAuthType = AuthType.getByName(config.getRemote().getAuthType());
+        defaultAuthType = config.getRemote().getAuthType();
```

Comment:
Seems like we can just remove this connector variable now?

Reply:
https://github.com/Konicai/Geyser/pull/1